feature_engine/features/politeness_v2.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+from features import feature_extraction
+
+def get_politeness_v2(df,on_column):
+    """
+    @Args:
+        The text dataframe
+    @Returns:
+        The dataframe after adding the politness v2 features
+    """
+
+    #extract the column headers by running the script for a random text. We sort the names of the features in alphabetical order. 
+
+    '''
+    TODO --- this code should be fixed stylistically; is there a cleaner way of doing this?
```

Comment:
See inline comment for cleaning this up stylistically! The 'hahaha' doesn't feel very readable!

Reply:
Changed it to using first row of df, but it seems like we need to call it on something at least once to get headers